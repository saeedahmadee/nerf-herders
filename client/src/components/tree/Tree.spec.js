import { shallowMount } from "@vue/test-utils";
import Tree from "./Tree.vue";

describe("Tree.vue", () => {
  const wrapper = shallowMount(Tree);
  it('should render correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
