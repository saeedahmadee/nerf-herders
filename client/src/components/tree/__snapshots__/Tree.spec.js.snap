// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tree.vue should render correctly 1`] = `
<div
  class="tree"
>
  <svg />
   
  <!---->
</div>
`;
